### PR TITLE
Integrating Cognito

### PR DESCRIPTION
**Pull request by salman0719**
## Features
- Typography & Button renovation
- Meta refactoring
- Loader component
- Toaster component
- Integrating Cognito authentication
  - Sign-up
  - Login
  - Code verification
  - Sign out
- Persisting user session
  - Temporarily utilizing React's useContext() to manage state in the client side (later to be replaced with Redux)

### CODE REVIEW

**Comment by salman0719** [comment]
@bhumkarpratik - Vercel is reporting that deployment failed. I'm guessing it could be something to do with `process.env` values. Anyways, can you check the details and let me know what the error is?

**Review by mikemike27 — COMMENTED** [review]
looks good!
one question though, the AuthFinishSignup doesn't do anything for now? eg the button is disabled and no onSubmit function. im guessing later it will redirect the user to the profile page? thanks.

**Comment by salman0719** [comment]
> looks good! one question though, the AuthFinishSignup doesn't do anything for now? eg the button is disabled and no onSubmit function. im guessing later it will redirect the user to the profile page? thanks.

Right, it will redirect the user to someplace else after saving the data - which will be done later on.

File: src/app/components/exploreMore/exploreMore.module.css
Comment (bhumkarpratik) [inline src/app/components/exploreMore/exploreMore.module.css:1]: Should we be deleting this file?

**Review by bhumkarpratik — APPROVED** [review]
LGTM

File: src/app/components/exploreMore/exploreMore.module.css
Comment (salman0719) [reply src/app/components/exploreMore/exploreMore.module.css:1]: Seems like this was an empty file!

**Review by bhumkarpratik — APPROVED** [review]

**Pull request merged**